docs/source/technical_tutorials/search/searching_edsc_gui.ipynb

```diff
@@ -4,59 +4,65 @@
    "cell_type": "markdown",
```

Comment:
<p>Looks good! The only thing I'd add here is "Search by spatial circle" in the second paragraph since there's also a function for that. Not a big deal, though 🙂</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/317/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I just added a circle png with the same hex color 😅